src/customizations/volto/components/manage/Blocks/Image/components/Image.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import cx from 'classnames';
+import '../style.less';
+
+const imageSizePicker = (url, size) => {
+  if (url) {
+    if (size === 'l') return `${url}/@@images/image`;
+    if (size === 'm') return `${url}/@@images/image/preview`;
+    if (size === 's') return `${url}/@@images/image/mini`;
+    return `${url}/@@images/image`;
+  }
+};
+
+const Image = ({ url, size, align, children }) => {
+  const styles = {
+    zIndex: 4,
+    position: 'relative',
+    backgroundImage: `url(${imageSizePicker(url, size)})`,
+    backgroundPosition: 'center',
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: 'cover',
+  };
+
+  const classes = cx({
+    imageElement: true,
+    'full-width': align === 'full',
+    large: size === 'l',
+    medium: size === 'm',
+    small: size === 's',
+  });
+
+  return (
+    <div className={classes} style={styles}>
```

Comment:
@andreiggr don't use inline styles, our website theme will be used by thematic websites as well
and they might need to tweak some CSS styles

Reply:
Hi @ichim-david 

Here inline styles are used to set the background image of that div. This way of setting the image is more ok for positioning the copyright properly.